app/invocation/invocation_action_input_node.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+interface Props {
+  node: any;
```

Comment:
nit: import your `Node` type from `invocation_action_card` and use it here -- or better yet, define the `Node` type here and have `invocation_action_card` import it from here instead (it probably makes more sense to define `Node` in this file because this component is the thing that renders the `Node`)